packages/framework/src/Command/CronCommand.php

```diff
@@ -247,6 +247,7 @@ private function chooseInstance(InputInterface $input, OutputInterface $output):
     private function getCronTimeZone(): DateTimeZone
     {
         $cronTimezone = $this->parameterBag->get('shopsys.cron_timezone');
```

Comment:
I would prefer this instead of `// @phpstan-ignore-next-line`
```suggestion
        /** @var string|null $cronTimezone */
        $cronTimezone = $this->parameterBag->get('shopsys.cron_timezone');
        $cronTimezone = $cronTimezone ?? date_default_timezone_get();
```

Reply:
good idea, thanks! fixed